java/client/src/main/java/glide/api/commands/StringBaseCommands.java

```diff
@@ -221,4 +221,30 @@ public interface StringBaseCommands {
      * }</pre>
      */
     CompletableFuture<Long> setrange(String key, int offset, String value);
+
+    /**
+     * Returns the substring of the string value stored at <code>key</code>, determined by the offsets
+     * <code>start</code> and <code>end</code> (both are inclusive). Negative offsets can be used in
+     * order to provide an offset starting from the end of the string. So <code>-1</code> means the
+     * last character, <code>-2</code> the penultimate and so forth.
```

Comment:
do we want to say what happens if end < start?

Reply:
Redis has a bug there (see IT)

Reply:
I saw that.  Better to just ignore it in the documentation...?

Reply:
All `*range` commands ignore such ranges - they don't reverse elements in the key being queried.
We don't document it in other commands.